### PR TITLE
mgr/dashboard: Add toBytes() method to FormatterService

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/shared/services/formatter.service.ts [hunk 1]
```diff
@@ -48,4 +50,51 @@ export class FormatterService {
...
+      'zb': Math.pow(base, 7),
+      'zib': Math.pow(base, 7),
+      'y': Math.pow(base, 8),
+      'yb': Math.pow(base, 8),
+      'yib': Math.pow(base, 8)
+    };
+    const m = RegExp('^(\\d+)\\s*(B|K(B|iB)?|M(B|iB)?|G(B|iB)?|T(B|iB)?|P(B|iB)?|' +
+      'E(B|iB)?|Z(B|iB)?|Y(B|iB)?)?$', 'i').exec(value);
+    if (m === null) {
+      return false;
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/services/formatter.service.ts]: How will this be used? Would it make more sense to return null? 
Could you give an example?

**Comment by votdev** [comment]
@tspmelo Your comment has been implemented.

**Comment by ricardoasmarques** [comment]
After this PR is merged, I'll adapt https://github.com/ceph/ceph/pull/20972/  accordingly.

**Comment by votdev** [comment]
jenkins retest this please

**Comment by votdev** [comment]
jenkins retest this please

**Pull request merged**